bib/internal/cntdnf/cntdnf.go

```diff
@@ -0,0 +1,48 @@
+package cntdnf
```

Comment:
Ondrej suggest to put this container package as containerSolver/dnfsolver and then have exec on container

Reply:
This is done now.

Reply:
cc @ondrejbudai